GOVERNANCE.rst

```diff
@@ -335,16 +346,51 @@ sktime uses a “consensus seeking” process for making decisions. The
+   must be satisfied:
+    - rejection by at least one core developer, and no acceptance by a core developer.
+    - rejection in the regular core developer meeting as part of the agenda.
+      This can happen only if no approvals or rejections are expresesd on a PR, see below.
+-  Formal statements on proposed changes must be located with the proposal on GitHub,
+   usually in the same pull request that contains the proposal, or the related issue.
+   Formal statements in this sense are approvals/rejections, and statements
+   about whether the proposal should be considered a "major change", as below.
 -  Approvals and rejections can be expressed as +1 and -1 comments,
-   respectively.
--  Core developers are expected to give reasonable time to others to
-   give their opinion on the pull request if they’re not confident
-   others would agree.
--  More important changes that impact the full project require a more
+   respectively, or any written comments
+   containing "I formally approve/reject" in reference to the proposal.
+   If the proposed change has the form of a PR, approval must be in the form of a GitHub PR approval.
+-  Major changes to the project require a more
    detailed analysis and a consensus that is both explicit and informed.
-   These changes require an :ref:`steps`.
+   A major change always require a worked out :ref:`steps`.
+   A change is major in the above sense
+   if at least three core developers explicitly state that the proposal is a major change
+   and requires especial consideration, in the location.
+-  Proposers are expected to give reasonable time for consideration, that is,
+   time and opportunity for core developers to review and
+   give their opinion on the contribution.
+   Ten working days excluding week-ends constitute "reasonable time" in the above sense.
+   For pull requests, the period resets at every new change made to the PR.
+   It starts only when all GitHub checks pass.
+-  If the "reasonable time" period elapses and no approval or rejection has been expressed on a PR,
+   the PR is scheduled for decision, at the top of the next regular core developer meeting's agenda
+   in which less than seven PR are already on the agenda. If no regular core developer meeting
+   is scheduled within the two calendar weeks after the period elapses,
+   the core developers must schedule one within those two weeks.
+-  Decision, in the core developer meeting, is by 2/3 majority of core developers in the meeting,
+   possibly including the proposer.
+   Decision is between approval or rejection. Approval can be subjected to minor corrections, which
+   requires an additional, simple majority consensus decision of core developers in the meeting.
+
+Approval or rejection as above count as "consensus".
+Lack of such consensus can arise under two conditions:
+- at least one approval and at least one rejection on GitHub
+- no 2/3 majority for either approval or rejection in the core developer meeting
```

Comment:
```suggestion
```

Reply:
For me, the main purpose of adding it to the agenda is for the PR to receive at least some feedback, at this stage we don't have to vote yet, voting is also already handled in stage 2, so I'd remove these changes here.

Reply:
Not against this, but please note that by simply removing this, the process is incomplete. How does it reach stage 2 in your preferred process? There is no link between moving it to the core dev meeting and stage 2.

Reply:
simplified